### ISSUE TITLE
Published crate contains benchmark reports

### ISSUE THREAD

**Issue by musicinmybrain** [EVaillant/lockfree-object-pool#6]
The GitHub repository contains a tree of criterion benchmark output at `benches/criterion/`, totaling about 10 MB.

These files appear in the published crate, making it much larger than necessary:

```
$ cargo publish --dry-run
$ tar -tzf target/package/lockfree-object-pool-0.1.5.crate
[…]
lockfree-object-pool-0.1.5/benches/bench_tools.rs
lockfree-object-pool-0.1.5/benches/criterion/allocation/crate 'object-pool'/base/benchmark.json
[…]
lockfree-object-pool-0.1.5/benches/criterion/reuse/spin_lock object poll/report/typical.svg
lockfree-object-pool-0.1.5/src/lib.rs
[…]
```

Please consider one of the following:

- Do not commit benchmark results to git (`git rm -r benches/criterion`)
- Leave benchmark results in git, but exclude them from the crate by adding `exclude = ["benches/criterion"]` to the `[package]` section of `Cargo.toml`

**Comment by EVaillant** [comment]
Thanks
I exclude result from crate

Commit 6f475979dc7e6a43190db5267680b73a1df88957 (HEAD -> master, origin/master, origin/HEAD)
Author: Etienne Vaillant <vaillant.etienne@gmail.com>
Date:   Sun May 19 15:21:24 2024 +0200

    Exclude benchmark reports from crate

**Comment by EVaillant** [comment]
new version 0.1.6 with fix 12 K (0.1.5 takes 2.26 M)

**Comment by musicinmybrain** [comment]
Awesome! Thanks for following up on this.
